auction-app-backend/src/main/java/com/example/auctionapp/entity/UserEntity.java

```diff
@@ -50,6 +53,16 @@ public class UserEntity implements UserDetails {
     @JoinColumn(name = "payment_info_id")
     private PaymentInfoEntity paymentInfoEntity;
 
+    @Column(name = "profile_picture")
+    private String profilePicture;
+
+    @Column(name = "dob")
+    @Temporal(TemporalType.DATE)
+    private LocalDate dob;
+
+    @Column(name="is_active")
+    private Boolean isActive;
```

Comment:
Do you know the difference between Boolean and boolean?

Reply:
boolean is a primitive type, while Boolean is like a wrapper around it which adds support to more methods